site/content/blog/2023-12-18-user-personas-poll/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- update
```

Comment:
`questionnaire` would be better than `update`

Reply:
Seems to have been missed somehow